variables.tf

```
@@ -8,12 +8,28 @@ variable "enforce_domain_name" {
 
 variable "enforce_group_id" {
   type        = string
-  description = "Enforce IAM group ID to bind your AWS account to"
+  description = "DEPRECATED: Please use 'enforce_group_ids'. Enforce IAM group ID to bind your AWS account to"
```

I think this additionally needs a default set to make the input variable optional

```suggestion
  description = "DEPRECATED: Please use 'enforce_group_ids'. Enforce IAM group ID to bind your AWS account to"
  default     = ""
```